tutorials/plot_sensors_time_frequency.py

```diff
@@ -70,12 +72,103 @@
 
 ax.plot(freqs, psds_mean, color='k')
 ax.fill_between(freqs, psds_mean - psds_std, psds_mean + psds_std,
-                color='k', alpha=.5)
+                color='k', alpha=.15, lw=0)
 ax.set(title='Multitaper PSD (gradiometers)', xlabel='Frequency',
        ylabel='Power Spectral Density (dB)')
 plt.show()
 
 ###############################################################################
+# To see the difference between :func:`psd_multitaper` and :func:`psd_welch`
+# - we are going to compare PSDs obtained with each function.
+ch_index = epochs.ch_names.index('MEG 2333')
```

Comment:
I agree, but combine is not only to make welch robust  in case of artifact-outliers - the distribution of power values will always be skewed (gamma-like, as shown in the example), so mean will be biased towards positive values. Making example in a follow-up PR will make the discussion clearer, I think.